api/v1alpha1/dataset_types.go

```diff
@@ -34,6 +34,8 @@ const (
 	FailedDatasetPhase DatasetPhase = "Failed"
 	// Not bound to runtime, can be deleted
 	NotBoundDatasetPhase DatasetPhase = "NotBound"
+	// Updated dataset, can't be released
+	UpdatedDatasetPhase DatasetPhase = "Updated"
```

Comment:
I think it should be "updating", to emphasis on ongoing

Reply:
Thanks, will use hump naming